README.md

```diff
@@ -68,23 +68,23 @@ roleRef:
-| spec.param.verbose                          | Increases checkup's log verbosity                                                                                 | False        | "true" / "false". Defaults to "false"                                                                     |
+| Key                                         | Description                                                                                                       | Is Mandatory | Remarks                                                                                                |
+|---------------------------------------------|-------------------------------------------------------------------------------------------------------------------|--------------|--------------------------------------------------------------------------------------------------------|
+| spec.timeout                                | How much time before the checkup will try to close itself                                                         | True         |                                                                                                        |
+| spec.param.networkAttachmentDefinitionName  | NetworkAttachmentDefinition name of the SR-IOV NICs connected                                                     | True         | Assumed to be in the same namespace                                                                    |
+| spec.param.trafficGeneratorRuntimeClassName | [Runtime Class](https://kubernetes.io/docs/concepts/containers/runtime-class/) the traffic generator pod will use | True         |                                                                                                        |
+| spec.param.trafficGeneratorImage            | Traffic generator's container image                                                                               | False        | Defaults to the U/S image https://quay.io/repository/kiagnose/kubevirt-dpdk-checkup-traffic-gen:latest |
+| spec.param.trafficGeneratorNodeSelector     | Node Name on which the traffic generator Pod will be scheduled to                                                 | False        | Assumed to be configured to Nodes that allow DPDK traffic                                              |
+| spec.param.trafficGeneratorPacketsPerSecond | Amount of packets per second in Millions                                                                          | False        | Defaults to 14                                                                                         |
+| spec.param.trafficGeneratorEastMacAddress   | MAC address of the NIC connected to the traffic generator pod/VM                                                  | False        | Defaults to a random MAC with prefix 0x50 and suffix 0x01                                              |
```

Comment:
I suggest to change the remarks format to:
Defaults to a random MAC address of the form: 50:xx:xx:xx:xx:01

Reply:
DONE